sonic-stormond/scripts/stormond

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env python3
+
+"""
+    stormond
+    Storage device Monitoring daemon for SONiC
+"""
+
+import os
+import signal
+import sys
+import threading
+import subprocess
+import shutil
+import json
+import time
+
+from sonic_py_common import daemon_base, device_info, syslogger
+from swsscommon import swsscommon
+from sonic_platform_base.sonic_storage.storage_devices import StorageDevices, BLKDEV_BASE_PATH
+
+#
+# Constants ====================================================================
+#
+
+SYSLOG_IDENTIFIER = "stormond"
+
+STORAGE_DEVICE_TABLE = "STORAGE_INFO"
+FSSTATS_SYNC_TIME_KEY = "FSSTATS_SYNC"
+
+# This directory binds to /host/pmon/stormond/ on the host
+FSIO_RW_JSON_FILE = "/usr/share/stormond/fsio-rw-stats.json"
+
+STORMOND_PERIODIC_STATEDB_SYNC_SECS = 3600 #one hour
+STORMOND_SYNC_TO_DISK_SECS = 86400 #one day
+
+STORAGEUTIL_LOAD_ERROR = 127
+
+log = syslogger.SysLogger(SYSLOG_IDENTIFIER)
```

Comment:
@assrinivasan can we move this inside daemon calss?

Reply:
Done in latest